sos/report/plugins/sunbeam.py

```diff
@@ -37,10 +48,90 @@ def setup(self):
             'sunbeam cluster list --format yaml',
         ])
 
+        sunbeam_user = self.get_option("sunbeam-user")
+        user_pwd = pwd.getpwnam(sunbeam_user)
+
+        if user_pwd:
+            sb_snap_homedir = f'{user_pwd.pw_dir}/snap/openstack/common'
+
+            self.add_copy_spec([
+                f"{sb_snap_homedir}/*.log",
+                f"{sb_snap_homedir}/etc/*/*.log",
+                f"{sb_snap_homedir}/logs/*.log",
+            ])
+
+            if self.get_option("juju-allow-login"):
+                self.exec_cmd(
+                    f'su - {sunbeam_user} -c "sunbeam utils juju-login"')
```

Comment:
This did not work with `runas` argument as this command has some interactive related items, and the only way to get this working was via `su`